vocab_files/observable_property_concepts/lithology-type.ttl

```diff
@@ -12,7 +12,7 @@ PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
     skos:exactMatch <http://linked.data.gov.au/def/tern-cv/e5b70249-76a8-4087-8ac9-588c7e29b944> ;
     skos:prefLabel "lithology type" ;
     schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/master/vocab_files/observable_property_concepts/lithology-type.ttl" ;
-    tern:hasCategoricalCollection <https://linked.data.gov.au/def/test/dawe-cv/46f50d33-eab0-4d00-9f84-58fd04017daf> ;
+    tern:hasCategoricalCollection <https://linked.data.gov.au/def/test/dawe-cv/1d50eb79-685f-45ea-84b4-627154eddede> ;
```

Comment:
What's the difference between this `lithology type` observable property and `coarse fragments lithology`? They both share the same categorical collection.

Reply:
Hi Edmond, after reading the protocol, I think they should be the same property:

Description from `plot soil description`:

> Assess and record Surface coarse fragments, including their Abundance, Size, Shape, Lithology,
> Strength and Alteration using the codes in Appendix J. Record mulitple if present. Refer to pages 139–
> 144, 209 and 214 of the YB for further descriptions.

Description from `soil pit characterisation`:

> If present, assess and record Coarse fragments, including their Abundance, Size, Shape, Lithology,
> Strength and Alteration using the codes in Appendix T. Record mulitple if present. Refer to pages 139–
> 144, 209 and 214 of the YB for further descriptions.

They are the same, both the lithology type of coarse fragments. 

However, all other information is different, you can run this query to see:
```
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
PREFIX dcterms: <http://purl.org/dc/terms/>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
select ?moduleCollectionLabel ?parameterLabel ?parameterDefinition ?valueType ?featureType ?parameterMethod ?parameterSource ?categoricalCollectionLabel ?api
{
    ?moduleCollectionURI skos:member ?parameterURI ;
                         skos:prefLabel ?moduleCollectionLabel .
    ?parameterURI skos:prefLabel ?parameterLabel ;
    				tern:valueType ?valueTypeURI .
    ?valueTypeURI rdfs:label ?valueType .
    optional {?parameterURI skos:definition ?parameterDefinition .}
    optional {
            ?parameterURI tern:hasFeatureType ?featureTypeURI . 
            service <https://graphdb.tern.org.au/repositories/tern_vocabs_core> {
        ?featureTypeURI skos:prefLabel ?featureType .
    }}
    optional {
        ?parameterURI tern:hasMethod ?parameterMethodURI .
        ?parameterMethodURI skos:prefLabel ?parameterMethod .
    }
    optional {
        ?parameterURI dcterms:source ?parameterSource .
    }
    optional {
        ?parameterURI tern:hasCategoricalCollection ?categoricalCollectionURI .
        ?categoricalCollectionURI skos:prefLabel ?categoricalCollectionLabel ;
                                  dcterms:source ?api .
    }
    filter (?parameterLabel in ('lithology type', 'coarse fragments lithology'))
}
order by ?parameterLabel
```

I don't know how Arun interpret this property, should we discuss with him?

Reply:
Yeah, we can confirm with Arun.

Reply:
Cool, I'll create another issue.